### ISSUE TITLE
Porting to tensorflow 2

### ISSUE THREAD

**Issue by mpdroid** [yangyanli/PointCNN#197]
Thank you for making this code available under MIT license. 

I tried running as-is with latest version of tensor-flow (2.0.0). However, there are major breaking changes between 1.6 and 2.0, code did not run at all. 
Moreover, 1.6 is also no longer available for download. So tried with version 1.15 and did not work either.

So have been attempting to port to TensorFlow 2.0. The work-in-progress is available at
[mpdroid/pointcnn_tf2](https://github.com/mpdroid/pointcnn_tf2)

So far have been able to define and run the model to the end. 

Have not been able to figure out how to implement the optimizer and loss functions, so using the out of the box loss and optimizer functions for now.

My model runs but prediction scores are hopeless.

Any suggestions from the original authors would be welcome.

**Comment by vhanded** [comment]
I tried 1.14, not working. 
The newest version that is working is 1.13, which I am using now.

**Comment by vhanded** [comment]
I tried to port to Keras, and able to achieve 88% val accuracy on modelnet, shy of 2% from the original implementation. I am suspecting is the input, but not sure. Code is super messy now.

**Comment by aldinorizaldy** [comment]
> Thank you for making this code available under MIT license.
> 
> I tried running as-is with latest version of tensor-flow (2.0.0). However, there are major breaking changes between 1.6 and 2.0, code did not run at all.
> Moreover, 1.6 is also no longer available for download. So tried with version 1.15 and did not work either.
> 
> So have been attempting to port to TensorFlow 2.0. The work-in-progress is available at
> [mpdroid/pointcnn_tf2](https://github.com/mpdroid/pointcnn_tf2)
> 
> So far have been able to define and run the model to the end.
> 
> Have not been able to figure out how to implement the optimizer and loss functions, so using the out-of-the-box loss and optimizer functions for now.
> 
> My model runs but prediction scores are hopeless.
> 
> 5/5 [==============================] - 217s 43s/step - loss: 5900.1611 - accuracy: 0.1079 - categorical_accuracy: 0.1079 - val_loss: 5365.4263 - val_accuracy: 0.3203 - val_categorical_accuracy: 0.3203
> 
> Any suggestions from the original authors would be welcome.

Hi, are you able to solve the prediction score?
